### DISCUSSION TITLE
How to print block programs

### DISCUSSION THREAD

**Discussion by msw-home** [pybricks/support#1728] [Q&A]
Moin, zusammen.
Gibt es die Möglichkeit, ein längeres Blockprogramm auszudrucken?

**Comment by laurensvalk** [comment]
You can click the camera icon to download a high resolution screenshot:

![image](https://github.com/user-attachments/assets/5dbf7023-8499-4d65-a383-31f0aa4bac8d)

You can print that directly, or crop and cut it as you like.

**Reply by msw-home** [reply]
Danke, super, genau das habe ich gesuchtI
